### PR TITLE
Add app endpoints to RPC API page in Lisk docs

### LINKED ISSUE #1857: Add `app` endpoints to RPC API page
### Description
The endpoints for the `app` namespace have been exposed recently and should be added to the documentation.

The issue depends on https://github.com/LiskHQ/lisk-sdk/issues/8875.

#### Affected branch / URL
`development`

#### Page location
`api/lisk-node-rpc.adoc`

### Additional resources
[application.ts](https://github.com/LiskHQ/lisk-sdk/blob/e990c678f6d43debfc9960e2a8f5f98ceaa9b87b/framework/src/application.ts#L375)

### PR DESCRIPTION
**Pull request by chris529**
Closes #1857

### CODE REVIEW

File: docs/modules/ROOT/pages/api/lisk-node-rpc.adoc [hunk 1]
```diff
@@ -44,6 +46,158 @@ The default RPC endpoints have been categorized based on various namespaces, as
 . <<txpoolEnd,Txpool>>
 
 
+=== App
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/api/lisk-node-rpc.adoc:49]:
```suggestion
=== `App`
```

File: docs/modules/ROOT/pages/api/lisk-node-rpc.adoc [hunk 2]
```diff
@@ -44,6 +46,158 @@ The default RPC endpoints have been categorized based on various namespaces, as
...
+==== `app_getRegisteredEndpoints`
+Lists all endpoints in the App.
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/api/lisk-node-rpc.adoc]:
```suggestion
Lists all endpoints registered to the Lisk application.
```

File: docs/modules/ROOT/pages/api/lisk-node-rpc.adoc [hunk 3]
```diff
@@ -44,6 +46,158 @@ The default RPC endpoints have been categorized based on various namespaces, as
...
+        "dynamicReward_getAnnualInflation",
+        "chainConnector_load",
+        "chainConnector_getSentCCUs",
+        "chainConnector_getAggregateCommits",
+        "chainConnector_getBlockHeaders",
+        "chainConnector_getCrossChainMessages",
+        "chainConnector_getLastSentCCM",
+        "chainConnector_getValidatorsInfoFromPreimage",
+        "chainConnector_authorize"
+]
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/api/lisk-node-rpc.adoc]: Improve indentation
```suggestion
[
  "app_getRegisteredActions",
  "app_getRegisteredEvents",
  "auth_getAuthAccount",
  "auth_isValidSignature",
  "auth_isValidNonce",
  "auth_getMultiSigRegMsgSchema",
  "auth_sortMultisignatureGroup",
  "auth_getMultiSigRegMsgTag",
  "validators_validateBLSKey",
  "validators_getValidator",
  "token_getBalances",
  "token_getBalance",
  "token_getTotalSupply",
  "token_getSupportedTokens",
  "token_isSupported",
  "token_getEscrowedAmounts",
  "token_getInitializationFees",
  "token_hasUserAccount",
  "token_hasEscrowAccount",
  "fee_getFeeTokenID",
  "fee_getMinFeePerByte",
  "interoperability_getRegistrationFee",
  "interoperability_getMinimumMessageFee",
  "interoperability_isChainIDAvailable",
  "interoperability_isChainNameAvailable",
  "interoperability_getChainAccount",
  "interoperability_getAllChainAccounts",
  "interoperability_getChannel",
  "interoperability_getOwnChainAccount",
  "interoperability_getTerminatedStateAccount",
  "interoperability_getTerminatedOutboxAccount",
  "interoperability_getChainValidators",
  "pos_getStaker",
  "pos_getValidator",
  "pos_getAllValidators",
  "pos_getLockedStakedAmount",
  "pos_getConstants",
  "pos_getPendingUnlocks",
  "pos_getPoSTokenID",
  "pos_getValidatorsByStake",
  "pos_getLockedReward",
  "pos_getClaimableRewards",
  "pos_getRegistrationFee",
  "pos_getExpectedSharedRewards",
  "random_isSeedRevealValid",
  "random_setHashOnion",
  "random_getHashOnionSeeds",
  "random_hasHashOnion",
  "random_getHashOnionUsage",
  "random_setHashOnionUsage",
  "dynamicReward_getExpectedValidatorRewards",
  "dynamicReward_getDefaultRewardAtHeight",
  "dynamicReward_getRewardTokenID",
  "dynamicReward_getAnnualInflation",
  "chainConnector_load",
  "chainConnector_getSentCCUs",
  "chainConnector_getAggregateCommits",
  "chainConnector_getBlockHeaders",
  "chainConnector_getCrossChainMessages",
  "chainConnector_getLastSentCCM",
  "chainConnector_getValidatorsInfoFromPreimage",
  "chainConnector_authorize"
]
```

File: docs/modules/ROOT/pages/api/lisk-node-rpc.adoc [hunk 4]
```diff
@@ -44,6 +46,158 @@ The default RPC endpoints have been categorized based on various namespaces, as
...
+----
+====
+
+==== `app_getRegisteredEvents`
+
+Lists all the events in the App.
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/api/lisk-node-rpc.adoc]:
```suggestion
Lists all the events registered to the Lisk application.
```

File: docs/modules/ROOT/pages/api/lisk-node-rpc.adoc [hunk 5]
```diff
@@ -44,6 +46,158 @@ The default RPC endpoints have been categorized based on various namespaces, as
...
+--
+[cols="1,1,1,3",options="header",stripes="hover"]
+Empty request body
+--
+cURL::
++
+--
+[source,json]
+----
+curl --location --request POST 'localhost:7887/rpc' \
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/api/lisk-node-rpc.adoc]: Just to be in sync with the rest of the document!
```suggestion
curl --location --request POST 'http://localhost:7887/rpc' \
```

**Review by ishantiw — APPROVED** [review]

File: docs/modules/ROOT/pages/api/lisk-node-rpc.adoc [hunk 6]
```diff
@@ -44,6 +46,158 @@ The default RPC endpoints have been categorized based on various namespaces, as
...
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/api/lisk-node-rpc.adoc]:
```suggestion
curl --location --request POST 'http://localhost:7887/rpc' \
```

File: docs/modules/ROOT/pages/api/lisk-node-rpc.adoc [hunk 7]
```diff
@@ -44,6 +46,158 @@ The default RPC endpoints have been categorized based on various namespaces, as
...
+.Response
+[%collapsible]
+====
+.Example output
+[source,json]
+----
+[
+        "app_ready",
+        "app_shutdown"
+]
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/api/lisk-node-rpc.adoc]:
```suggestion
[
  "app_ready",
  "app_shutdown"
]
```

**Review by TalhaMaliktz — CHANGES_REQUESTED** [review]
Looks good already! Minor cosmetic touch-ups are required.

File: docs/modules/ROOT/pages/api/lisk-node-rpc.adoc (hunk 1 above)
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/api/lisk-node-rpc.adoc:49]: It seems that there is an inconsistency with the level-3 headings as some of them are wrapped in monospace whilst others aren't. Please remove monospace from all level-3 headings.

Comment (chris529) [reply docs/modules/ROOT/pages/api/lisk-node-rpc.adoc:49]: Done 👍

**Review by TalhaMaliktz — APPROVED** [review]
Thanks! LGTM 👍🏼

**Pull request merged**